### PR TITLE
function to find the types of arg and sum

### PR DESCRIPTION
**Pull request by selvarajsathish**

### CODE REVIEW

File: functions/sum-of-inputs.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+def sum_of_inputs(a,b):           # To calculate the sume of given value.
+		c=a+b
```
Comment (gnurenga) [inline functions/sum-of-inputs.py]: Remove this function

File: functions/sum-of-inputs.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+		return c
+
+def find_type_arg(input1,input2): # To find the type of arguments given.
+	x,y=type(input1),type(input2)
+	if (x!= y):
+		return -1
+
```
Comment (gnurenga) [inline functions/sum-of-inputs.py:22]: add else part and return  input1 + input2

File: functions/sum-of-inputs.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+if __name__ == '__main__':
+	a,b=input("enter a two values to be sum :")		
+	ret=find_type_arg(a,b)        # called function to find type of arguments given by user.
+	if ret == -1:
+		print 'Two inputs should be either integer or string.'
+	else:
+		print sum_of_inputs(a,b)  # called the fucntion to calculate the sume of given value.
```
Comment (gnurenga) [inline functions/sum-of-inputs.py]: * remove sum_of_input
* print  "sum of input 1 and input2 is", ret

File: functions/sum-of-inputs.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+
+def sum_of_inputs(a,b):           # To calculate the sume of given value.
+		c=a+b
+		return c
+
+def find_type_arg(input1,input2): # To find the type of arguments given.
+	x,y=type(input1),type(input2)
```
Comment (gnurenga) [inline functions/sum-of-inputs.py]: you should not compare two values, instead you should check both arguments or integer or string. Suppose if i pass both argument other than int or string then it should  return -1.

File: functions/sum-of-inputs.py (hunk 1 above)
Comment (gnurenga) [reply functions/sum-of-inputs.py]: No need to remove the function

File: functions/sum-of-inputs.py (hunk 3 above)
Comment (gnurenga) [reply functions/sum-of-inputs.py]: Keep it same as before, I was wrong.

File: functions/find-type.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+def find_type_arg():
+	arg=input("enter a arguement to find its type :")
```
Comment (gnurenga) [inline functions/find-type.py]: Get the input in main code and pass the argument.
def find_type_arg(value):
      print type(value)

File: functions/sum-of-inputs.py [hunk 6]
```diff
@@ -0,0 +1,14 @@
+def sum_of_inputs(x,y):                 # To calculate the sume of given value.	
+	if (type(x) != int or type(y) !=int) and (type(x) != str or type(y) != str):
+		return -1
+	else:
+		c=x+y
+		return c
```
Comment (gnurenga) [inline functions/sum-of-inputs.py]: return x + y

File: functions/sum-of-inputs.py [hunk 7]
```diff
@@ -0,0 +1,14 @@
+def sum_of_inputs(x,y):                 # To calculate the sume of given value.	
+	if (type(x) != int or type(y) !=int) and (type(x) != str or type(y) != str):
```
Comment (gnurenga) [inline functions/sum-of-inputs.py:18]: did you tested this code. it wont work

Comment (selvarajsathish) [reply functions/sum-of-inputs.py:18]: I tested it was working..

Comment (gnurenga) [reply functions/sum-of-inputs.py:18]: ok.

File: functions/find-type.py [hunk 8]
```diff
@@ -0,0 +1,6 @@
+def find_type_arg(value): # Fuction to find type of given value.	
+	print type(value)
```
Comment (gnurenga) [inline functions/find-type.py:15]: Use function doc string based on this reference https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

File: functions/sum-of-inputs.py [hunk 9]
```diff
@@ -0,0 +1,14 @@
+def sum_of_inputs(x,y):                 # To calculate the sume of given value.
```
Comment (gnurenga) [inline functions/sum-of-inputs.py]: Use function doc string based on this reference https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

File: functions/find-type.py [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+        
+    Example:
+    	value  | return 
+    	2      | <type 'int'>
+
+    """	
+	print type(value)
+
+if __name__ == '__main__':
+	"""main function to get user input and call find_type_arg fuction to print type of value.
```
Comment (gnurenga) [inline functions/find-type.py:18]: You can remove arg in this conditional statement. Just you can have comment what you are doing in this condition

File: functions/sum-of-inputs.py [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+	if (type(x) != int or type(y) !=int) and (type(x) != str or type(y) != str):
+		return -1
+	else:
+		return x+y		
+
+if __name__ == '__main__':
+	"""main function to get user input and call sum_of_inputs fuction to perform sum and return value.
+
+    Args:
+       a (int or str): The first parameter.
```
Comment (gnurenga) [inline functions/sum-of-inputs.py]: Funtion doct string is not suitable to comment in this conditional statement, instead you can explain what you are doing in this block.

**Review by gnurenga — APPROVED** [review]

**Pull request merged**